README.md

```diff
@@ -1,11 +1,6 @@
 # Ekostraz
 Wrocław Serverless Hackathon application
 
-| Frontend  | Backend |
```

Comment:
Czemu w zasadzie to usuwamy zamiast naprawić pipeline'y? 😁

Reply:
Pipeline dodamy na devopsach powiązanych z kontem na którym będzie wdrożona apka. To są chyba devopsy Patryka do których nie mamy dostępu/uprawnień

Reply:
Oki :)